src/aiofiles/os.pyi

```diff
@@ -0,0 +1,220 @@
+import sys
+from asyncio.events import AbstractEventLoop
+from collections.abc import Sequence
+from concurrent.futures import Executor
+from os import _ScandirIterator, stat_result
+from typing import AnyStr, overload
+
+from _typeshed import (
+    BytesPath,
+    FileDescriptorOrPath,
+    GenericPath,
+    ReadableBuffer,
+    StrOrBytesPath,
+    StrPath,
+)
+
+from aiofiles import ospath
+from aiofiles.ospath import wrap as wrap
+
+__all__ = [
+    "path",
+    "stat",
+    "rename",
+    "renames",
+    "replace",
+    "remove",
+    "unlink",
+    "mkdir",
+    "makedirs",
+    "rmdir",
+    "removedirs",
+    "link",
+    "symlink",
+    "readlink",
+    "listdir",
+    "scandir",
+    "access",
+    "wrap",
+    "getcwd",
+]
+
+if sys.platform != "win32":
+    __all__ += ["statvfs", "sendfile"]
```

Comment:
You can place this line after [this one](https://github.com/Tinche/aiofiles/pull/222/files#diff-67253a8173e8ad73d8d0a0e6a526d03efa4f6ec5f8166138b3ca079664705d82R220) and this if can be removed.